### DISCUSSION TITLE
FDE's Wayland Window Manager Overriding Linux X Window Management

### DISCUSSION THREAD

**Discussion by superligen** [openfde/openfde#129] [General]
On Linux, after FDE starts, applications such as Terminator appear on the FDE desktop and are not managed by the Linux window manager. It seems that FDE's Wayland window manager takes precedence, causing minimized windows to become inaccessible as they are hidden behind the FDE desktop.

**Comment by warlice** [comment]
fixed， X11 windows will start in another workspace。 so it won't hidden behind the FDE desktop
